### PR TITLE
Show user settings screen on registration

### PR DESCRIPTION
**Pull request by dbkr**
instead of the PostRegister stuff.

### CODE REVIEW

File: src/components/structures/MatrixChat.js [hunk 1]
```diff
@@ -697,7 +697,14 @@ module.exports = React.createClass({
     onRegistered: function(credentials) {
         this.onLoggedIn(credentials);
         // do post-registration stuff
-        this.showScreen("post_registration");
+        // This now goes straight to user settings
+        // We use _setPage since if we wait for
+        // showScreen to do the dispatch loop,
+        // we'll see the page type is still unset
```
Comment (ara4n) [inline src/components/structures/MatrixChat.js]: why would the page type still be unset?

**Comment by ara4n** [comment]
lgtm other than the question, which is more for my own edification

File: src/components/structures/MatrixChat.js (hunk 1 above)
Comment (dbkr) [inline src/components/structures/MatrixChat.js]: Does this comment make more sense?

**Comment by ara4n** [comment]
yup, thanks, lgtm

**Pull request merged**